stroeer/content/v1/content_service.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+import "stroeer/page/section/v1/section_page.proto";
+
+package stroeer.content.v1;
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/content/v1;content";
+option java_multiple_files = true;
+option java_package = "de.stroeer.content.v1";
+
+service ContentService {
```

Comment:
As our `ArticlePageService` and `SectionPageService`, this should either be a `FragmentService` (or `Partial(s)Service`) because `Content` is too broad.

Another better fitting name would be `StageService`